apps/charterafrica/src/components/Tool/OrgThemeAndOperatingCountries.js

```diff
@@ -0,0 +1,68 @@
+import { RichTypography } from "@commons-ui/core";
+import { Figure } from "@commons-ui/next";
+import { Box, Button } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+const OrgThemeAndOperatingCountries = React.forwardRef(
+  function OrgThemeAndOperatingCountries(props, ref) {
+    const { sx, organisation, theme, operatingCountries } = props;
+    const countries = operatingCountries?.join(", ");
+    if (!organisation) {
+      return null;
+    }
+    return (
+      <Box sx={sx} ref={ref}>
+        <Figure
+          ImageProps={{
+            alt: organisation.name,
+            src: organisation.avatarUrl,
+          }}
+          sx={{
+            height: 50,
+            width: "auto",
+          }}
+        />
+        <RichTypography
+          textAlign="left"
+          color="neutral.dark"
+          sx={{ fontWeight: 400 }}
+          variant="h4"
+        >
+          {organisation.name}
+        </RichTypography>
+        {theme ? (
+          <Button
```

Comment:
What is supposed to happen when a user clicks this button?

Reply:
Shouldn't actually be button @kilemensi that's why I set component as `div`